### ISSUE TITLE
Question are not programmed as required in form (WCAG 3.3.3) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#207]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 3: Observation Print
Lesson 4: Manual QA
Lesson 5: Reports

**Description:**
The page is progressively revealed as the user completes questions. As such the user is required to answer questions in order to completely submit the form. But none of the questions are programmatically required.

Questions should be set to required so that the user is informed they need to answer questions to proceed.
![image](https://user-images.githubusercontent.com/93537804/229823168-40773513-f652-4801-8140-14624caa275c.png)
*******************************************************************************
